code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -17,6 +17,10 @@
 		do_sparks(rand(5, 9), FALSE, src)
 		playsound(T, 'sound/effects/bang.ogg', 100, TRUE)
 		new /obj/effect/dummy/lighting_obj(T, light_color, range + 2, light_power, light_time)
+	// Blob damage
+	for(var/obj/structure/blob/B in hear(range + 1, T))
+		var/damage = round(30 / (get_dist(B, T) + 1))
+		B.take_damage(damage, BURN, "melee", FALSE)
```

Comment:
```suggestion
		// Blob damage
		for(var/obj/structure/blob/B in hear(range + 1, T))
			var/damage = round(30 / (get_dist(B, T) + 1))
			B.take_damage(damage, BURN, "melee", FALSE)
```
Include this in the `if(T)` check since it uses range on the turf itself.